package me.santio.utils.database.exceptions

class InvalidEntityException(reason: String? = null): RuntimeException(reason)